bin/autoformat

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-rubocop -a -c rubocop.yml
+rubocop -A -c .rubocop.yml
```

Comment:
This has been wrong for a long time and nobody noticed. Should we stop generating this file?